spec/factories/claimant.rb

```
@@ -4,7 +4,7 @@
   factory :claimant do
     sequence(:decision_review_id)
     sequence(:participant_id)
-    decision_review_type { "Appeal" }
+    association :decision_review, factory: :appeal
```

This change in `master` makes the validation "work". It seemed like there was a really deep exception that was being swallowed because an appeal didn't exist for some claimants, which was necessary for validation.